sklearn/linear_model/sgd_fast.pyx

```diff
@@ -700,7 +728,7 @@ def _plain_sgd(np.ndarray[double, ndim=1, mode='c'] weights,
 
     w.reset_wscale()
 
```

Comment:
You could disable the convergence warning by passing `tol=0` as a local variable only when called from `partial_fit` while passing `tol=self.tol` when called from `fit`.